### PR TITLE
make table rows clickable

### PR DESCRIPTION
**Pull request by LawalRahman**
- Make table rows clickable

### CODE REVIEW

File: components/admin/data-table/data-table.composition.tsx [hunk 1]
```diff
@@ -2,9 +2,23 @@ import React from "react"; //
...
+    name: "Duah",
+    address: "bn-34343",
+    title: "Ninja 3",
+    onClick: () => console.log("Duah"),
+  },
+  {
+    name: "Rahman",
+    address: "bx-34343",
+    title: "Ninja 5",
+    onClick: () => console.log("Rahman"),
```
Comment (nanaduah1) [inline components/admin/data-table/data-table.composition.tsx]: Why is the click defined on the data? If I load data from the server how will it have a click?

**Review by nanaduah1 — CHANGES_REQUESTED** [review]

File: components/admin/data-table/data-table.composition.tsx (hunk 1 above)
Comment (LawalRahman) [reply components/admin/data-table/data-table.composition.tsx]: If we want to assign a click event to it, I think the best place to do that is before passing the data to the component. That way, we can control what the click even does without having to change the component at all. @nanaduah1

Comment (nanaduah1) [reply components/admin/data-table/data-table.composition.tsx]: Is the user clicking the table or the data? If we have 2 tables using the same data we add 2 functions on the data right?

Comment (LawalRahman) [reply components/admin/data-table/data-table.composition.tsx]: The data

Comment (nanaduah1) [reply components/admin/data-table/data-table.composition.tsx]: The data is not a UI element so how does a user click it? The user can only click a UI element that represents the data.
Assuming we have 2 tables showing same data and different format.

But each table does something different when the row is clicked. How would you handle that if the onClick is defined on the data?

**Review by nanaduah1 — APPROVED** [review]

**Pull request merged**